scst-store/amr/data-model-and-concepts.hbs.md

```diff
@@ -76,6 +76,12 @@ files when you generate a new project. Observer sends this information to the
 Cloud Event Handler to store `AppAcceleratorRuns`. See [cloud
 events](cloudevents.hbs.md).
 
+Each `AppAcceleratorRun` data entry has an unique `guid`. It includes information 
```

Comment:
[This arbitrary website](https://www.scribbr.com/frequently-asked-questions/an-or-a-unique/) says that it's supposed be `a unique` 🤷‍♀️